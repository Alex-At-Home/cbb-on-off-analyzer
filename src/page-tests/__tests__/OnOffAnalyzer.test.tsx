import React from 'react';
import OnOffAnalyzerPage from '../../pages/OnOffAnalyzer';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("OnOffAnalyzerPage", () => {
  test("OnOffAnalyzerPage - should create snapshot", () => {
    const wrapper = shallow(<OnOffAnalyzerPage/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
