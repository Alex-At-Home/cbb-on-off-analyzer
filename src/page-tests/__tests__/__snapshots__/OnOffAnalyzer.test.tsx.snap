// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnOffAnalyzerPage OnOffAnalyzerPage - should create snapshot 1`] = `
<Container
  fluid={false}
>
  <Row
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <h3>
        CBB On/Off Analysis Tool
         
        <span
          className="badge badge-pill badge-info"
        >
          BETA!
        </span>
      </h3>
    </Col>
  </Row>
  <Row
    noGutters={false}
  >
    <HeaderBar
      common={Object {}}
      thisPage="game-"
    />
  </Row>
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      minimizeMargin={false}
      summary="On/Off: 2024/25  (M): on:'', auto-off, base:''"
      title="Team and Game Filter"
    >
      <GameFilter
        forceReload1Up={0}
        onChangeState={[Function]}
        onStats={[Function]}
        startingState={Object {}}
      />
    </GenericCollapsibleCard>
  </Row>
  <InternalNavBarInRow
    refs={
      Object {
        "Compare Lineups": Object {
          "current": null,
        },
        "Player Analysis": Object {
          "current": null,
        },
        "Team Analysis": Object {
          "current": null,
        },
        "Top": Object {
          "current": null,
        },
      }
    }
  />
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      helpLink="https://hoop-explorer.blogspot.com/2019/11/fun-with-college-basketball-onoff.html"
      minimizeMargin={true}
      title="Team Analysis"
    >
      <TeamStatsTable
        dataEvent={
          Object {
            "lineupStats": Array [],
            "rosterStats": Object {
              "baseline": Array [],
              "global": Array [],
              "off": Array [],
              "on": Array [],
            },
            "shotStats": Object {
              "baseline": Object {
                "def": Object {},
                "off": Object {},
              },
              "off": Object {
                "def": Object {},
                "off": Object {},
              },
              "on": Object {
                "def": Object {},
                "off": Object {},
              },
            },
            "teamStats": Object {
              "baseline": Object {
                "doc_count": 0,
              },
              "global": Object {
                "doc_count": 0,
              },
              "off": Object {
                "doc_count": 0,
              },
              "on": Object {
                "doc_count": 0,
              },
            },
          }
        }
        gameFilterParams={Object {}}
        onChangeState={[Function]}
      />
    </GenericCollapsibleCard>
  </Row>
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      helpLink="https://hoop-explorer.blogspot.com/2019/11/fun-with-college-basketball-onoff.html"
      minimizeMargin={true}
      title="Individual Analysis"
    >
      <RosterStatsTable
        dataEvent={
          Object {
            "lineupStats": Array [],
            "rosterStats": Object {
              "baseline": Array [],
              "global": Array [],
              "off": Array [],
              "on": Array [],
            },
            "shotStats": Object {
              "baseline": Object {
                "def": Object {},
                "off": Object {},
              },
              "off": Object {
                "def": Object {},
                "off": Object {},
              },
              "on": Object {
                "def": Object {},
                "off": Object {},
              },
            },
            "teamStats": Object {
              "baseline": Object {
                "doc_count": 0,
              },
              "global": Object {
                "doc_count": 0,
              },
              "off": Object {
                "doc_count": 0,
              },
              "on": Object {
                "doc_count": 0,
              },
            },
          }
        }
        gameFilterParams={Object {}}
        onChangeState={[Function]}
      />
    </GenericCollapsibleCard>
  </Row>
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      helpLink="https://hoop-explorer.blogspot.com/2019/11/fun-with-college-basketball-onoff.html"
      minimizeMargin={false}
      title="Lineup Comparison"
    >
      <RosterCompareTable
        gameFilterParams={Object {}}
        rosterCompareStats={
          Object {
            "baseline": Object {},
            "off": Object {},
            "on": Object {},
          }
        }
      />
    </GenericCollapsibleCard>
  </Row>
  <Footer
    server="server"
  />
</Container>
`;
